projects/packages/changelogger/src/AddCommand.php

```diff
@@ -285,7 +286,7 @@ function ( $v ) {
 					return 1;
 				}
 			} else {
-				if ( null === $entry ) {
+				if ( null === $entry && $comment === '' ) {
 					$output->writeln( '<error>Entry must be specified in non-interactive mode.</>' );
```

Comment:
This error message is kind of misleading now. When significance is patch and a comment is specified, the entry doesn't have to be specified.